app/forms/providers/proceeding_merits_task/parental_responsibilities_form.rb

```diff
@@ -0,0 +1,26 @@
+module Providers
+  module ProceedingMeritsTask
+    class ParentalResponsibilitiesForm < BaseForm
+      form_for Proceeding
+
+      attr_accessor :relationship_to_child
+
+      validates :relationship_to_child,
+                inclusion: {
+                  in: %w[court_order parental_responsibility_agreement false],
+                  message: I18n.t("providers.proceeding_merits_task.relationship_to_child.parental_responsibilities.error"),
```

Comment:
Is it possible to remove this key:value, store the error message in our activemodel locales file, and rely on rails magic and activemodel attributes to get the value?

Reply:
I've discussed with Colin (who followed a similar pattern in his previous ticket) and it's not obvious to us where it should go. Colin said he's happy to spend some time investigating it now but is it ok for this to progress for now, as it's holding up a few tickets?